tests/test_cli.py

```diff
@@ -321,6 +321,54 @@ def test_mlflow_gc_file_store_older_than(file_store):
     assert len(runs) == 0
 
 
+@pytest.mark.parametrize("get_store_details", ["file_store", "sqlite_store"])
+def test_mlflow_gc_experiments(get_store_details, request):
```

Comment:
Can we also add some `search_runs()` validations to this test case ensuring that the experiment's runs have been deleted as well?